Content.Server/Atmos/Reactions/N2ODecompositionReaction.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Server.Atmos.EntitySystems;
+using Content.Shared.Atmos;
+using JetBrains.Annotations;
+
+namespace Content.Server.Atmos.Reactions;
+
+/// <summary>
+///     Decomposes Nitrous Oxide into Nitrogen and Oxygen.
+/// </summary>
+[UsedImplicitly]
+public sealed partial class N2ODecompositionReaction : IGasReactionEffect
+{
+    public ReactionResult React(GasMixture mixture, IGasMixtureHolder? holder, AtmosphereSystem atmosphereSystem, float heatScale)
+    {
+        var cacheN2O = mixture.GetMoles(Gas.NitrousOxide);
+
+        var burnedFuel = cacheN2O * Atmospherics.N2ODecompositionRate * (mixture.Temperature - Atmospherics.N2ODecompositionMinScaleTemp) * (mixture.Temperature - Atmospherics.N2ODecompositionMaxScaleTemp) / Atmospherics.N2ODecompositionScaleDivisor;
+
+        //if (burnedFuel <= 0 || cacheN2O - burnedFuel < 0) return ReactionResult.NoReaction;
+
+        mixture.AdjustMoles(Gas.NitrousOxide, burnedFuel * Atmospherics.N2ODecompositionRate);
+        mixture.AdjustMoles(Gas.Nitrogen, -burnedFuel * Atmospherics.N2ODecompositionRate);
+        mixture.AdjustMoles(Gas.Oxygen, (-burnedFuel / 2) * Atmospherics.N2ODecompositionRate);
```

Comment:
`Atmospherics.N2ODecompositionRate` already goes into the `burnedFuel` calculation, so why multiply it again? Why not just square `Atmospherics.N2ODecompositionRate`?